.changeset/pretty-terms-repair.md

```diff
@@ -0,0 +1,5 @@
+---
+"live-mobile": minor
```

Comment:
I'd say patch, no?

Reply:
Why? It's not a bugfix it's the v2 of a feature